index.js

```diff
@@ -258,6 +266,18 @@ module.exports = {
     return this._coveredAddon;
   },
 
+  /**
+   * Find the module (if any) that's being covered.
+   * @returns {Module} the module under test
+   */
+  _findCoveredModule: function() {
+    if (!this._coveredModule) {
+      this._coveredModule = require(this.project.findAddonByName(this.project.pkg.name).root);
```

Comment:
We should definitely not be requiring an addon's entry point directly and making assumptions about it.